src/me/corriekay/pokegoutil/utils/pokemon/PokemonCalculationUtils.java

```diff
@@ -72,7 +72,11 @@ public static double ivRating(final Pokemon p) {
      */
     public static double dpsForMove(final Pokemon p, final boolean primary) {
         final PokemonMove move = primary ? p.getMove1() : p.getMove2();
-        return dpsForMove(p.getPokemonId(), move, primary);
+        double dps = 0.0;
```

Comment:
The 'dpsForMove' changes is doing this verification already. There's no need for this verification